content/07.pathogenesis.md

```diff
@@ -117,12 +116,12 @@ The luminal (interior) surface of the endothelium is lined with glycocalyx, a ne
 The glycocalyx varies in thickness from 0.5 microns in the capillaries to 4.5 microns in the carotid arteries and forms a meshwork that localizes both endothelial- and plasma-derived signals to the inner vessel wall [@doi:10.1007/s00424-007-0212-8].
 Heparan sulfate is the dominant proteoglycan in the glycocalyx, representing 50-90% of glycocalyx proteoglycan content [@doi:10/bp82dx].
 The SARS-CoV-2 spike protein can bind directly to heparan sulfate, which serves in part as a scaffolding molecule to facilitate ACE2 binding and entry into endothelial cells [@doi:10.1016/j.cell.2020.09.033].
-A heparan sulfate binding site has also been identified near the ACE2 binding site on the viral RBD, and modeling has suggested that heparan sulfate binding yields an open conformation that facilitates binding to ACE2 on the cell surface [@doi:10.1016/j.cell.2020.09.033].
+A heparan sulfate binding site has also been identified near the ACE2 binding site on the viral receptor binding domain (RBD), and modeling has suggested that heparan sulfate binding yields an open conformation that facilitates binding to ACE2 on the cell surface [@doi:10.1016/j.cell.2020.09.033].
 Degrading or removing heparan sulfate was associated with decreased binding [@doi:10.1016/j.cell.2020.09.033].
 Heparan sulfate may also interact with the S1/S2 proteolytic cleavage site and other binding sites to promote binding affinity [@doi:10.1101/2020.05.10.087288].
 Notably, treatment with soluble heparan sulfate or even heparin (a commonly used anti-coagulant and vasodilator that is similar in structure to heparan sulfate [@doi:10.1007/978-3-642-23056-1_8]) potently blocked spike protein binding and viral infection [@doi:10.1016/j.cell.2020.09.033].
 This finding is particularly interesting because degradation of heparan sulfate in the glycocalyx has previously been identified as an important contributor to ARDS and sepsis [@doi:10.1016/bs.ctm.2018.08.005], two common and severe outcomes of COVID-19, and suggests that heparan sulfate could be a target for pharmaceutical inhibition of cell entry by SARS-CoV-2 [@doi:10.1038/s41421-020-00222-5; @doi:10.1016/j.ebiom.2020.102969; @doi:10.1016/j.amjcard.2020.08.005; @doi:10.3390/nu13020328; @doi:10.3389/fimmu.2021.607306].
-Together, this evidence suggests that heparan sulfate can serve as an important adhesion molecule for SARS-CoV-2 cell entry and may represent a therapeutic target.
+Together, this evidence suggests that heparan sulfate can serve as an important adhesion molecule for SARS-CoV-2 cell entry and may represent a therapeutic target. <!-- It's slightly surprising to see the case made for heparan sulfate here because it doesn't come up at all (right?) in the therapeutics manuscript. Herapin is noted as SOC in an early trial, but that's all I recall. -->
```

Comment:
```suggestion
Together, this evidence suggests that heparan sulfate can serve as an important adhesion molecule for SARS-CoV-2 cell entry.
It may represent a therapeutic target but has not been pursued as much as other candidate targets [@individual-pharmaceuticals].
```
How is this phrasing?  I think it is okay to point out the therapeutic hypothesis here, but I didn't want to suggest it is one of the leading candidates or even one that has been studied all that much.

Reply:
That is a great way to unify them!